deploy/alerts/terraform/roles/onafterstart/tasks/main.yml

```diff
@@ -0,0 +1,87 @@
+---
+
+# This play require the following variable set
+#   newrelic_personal_api_key: "<from user config newrelic credential>"
+#   newrelic_admin_api_key: "<from user config newrelic credential>"
+#   newrelic_account_id: "<from user config newrelic credential>"
+#   newrelic_region: "<from user config newrelic credential>"
+#   service_display_name: "The entity display_name to create alert for"
+# The fields below are optional and can be set through the deploy config params
+#   s3_bucketname_tfstate: "<an AWS S3 bucket name to use for the terraform state file>"
+#   alert_duration: the number of minutes for a condition before reacting
+#   alert_response_time_threshold: the maximum number of seconds for processing requests before alerting
+#   alert_throughput_threshold: the minimum number of requests per minutes before alerting
+#   alert_error_percentage_threshold: the maximum percentage of requests having an error before alerting
+#   alert_cpu_threshold: the maximum percentage for the host CPU before alerting
+# Other optional fields:
+#   aws_access_key: the aws api key when an S3 bucket is used
+#   aws_secret_key: the aws secret key when an S3 bucket is used
+#   aws_region: the aws region key when an S3 bucket is used
+
+- debug:
+    msg: "Deploying alerts for {{ service_display_name }} using Terraform for Golden Signals policy"
+
+- fail:
+    msg: "newrelic_personal_api_key is required. Create this entry in your user config file"
+  when: newrelic_personal_api_key is not defined
+- fail:
+    msg: "newrelic_admin_api_key is required. Create this entry in your user config file"
+  when: newrelic_admin_api_key is not defined
+- fail:
+    msg: "newrelic_account_id is required. Create this entry in your user config file"
+  when: newrelic_account_id is not defined
+- fail:
+    msg: "newrelic_region is required. Create this entry in your user config file"
+  when: newrelic_region is not defined
```

Comment:
I wonder if you might want to use the key names that are in the user config file.  For example I get an error that says:
'newrelic_personal_api_key is required....'
But the key I need to add is called 'nrPersonalApiKey'.

Reply:
Sure